README.md

```diff
@@ -638,9 +638,9 @@ European Conf. Computer Vision (ECCV), 2020. [Suppl. Mat.](https://www.ecva.net/
 Sensors 2021, 21, 1150. [PDF](https://www.mdpi.com/1424-8220/21/4/1150/pdf)
 - [Paredes-Valles et al., CVPR 2021](#ParedesValles21cvpr),  
 *Back to Event Basics: Self-Supervised Learning of Image Reconstruction for Event Cameras via Photometric Constancy.*
-- <a name="ParedesValles21arxiv"></a>Paredes-Valles, F., Hagenaars, J. J., de Croon, G. C. H. E.,  
-*[Self-Supervised Learning of Event-Based Optical Flow with Spiking Neural Networks](https://arxiv.org/abs/2106.01862)*,  
-arXiv:2106.01862, 2021.
+- <a name="ParedesValles21nips"></a>Paredes-Valles, F., Hagenaars, J. J., de Croon, G. C. H. E.,  
```

Comment:
nips -> neurips

Reply:
Could you swap the order of the authors, to be consistent with the paper? thanks.

Reply:
fixed!